### PR TITLE
Revamp gitian building guides

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
Signed-off-by: pasta <pasta@dashboost.org>

### CODE REVIEW

File: doc/gitian-building.md [hunk 1]
```diff
@@ -1,494 +1,129 @@
...
+git push --set-upstream $NAME $VERSION-not-codesigned
 ```
-/etc/apt-cacher-ng/acng.conf
-[...]
-Offlinemode: 1
-[...]
 
-service apt-cacher-ng restart
-```
+You can also mail the files to Wladimir (laanwj@gmail.com) and he will commit them.
```
Comment (thephez) [inline doc/gitian-building.md:113]: I don't think Wladimir is going to be very helpful for us :-)

File: doc/gitian-building.md [hunk 2]
```diff
@@ -1,494 +1,127 @@
...
-    # (must return OK)
-
-Then start the VM. On the first launch you will be asked for a CD or DVD image. Choose the downloaded ISO.
-
-![](gitian-building/select_startup_disk.png)
-
-Installing Debian
+MacOS code signing
 ------------------
+In order to sign builds for macOS, you need to download the free SDK and extract a file. The steps are described [here](./gitian-building/gitian-building-mac-os-sdk.md). Alternatively, you can skip the macOS build by adding `--os=lw` below.
```
Comment (thephez) [inline doc/gitian-building.md:53]: Don't think the `./` is needed
```suggestion
In order to sign builds for macOS, you need to download the free SDK and extract a file. The steps are described [here](gitian-building/gitian-building-mac-os-sdk.md). Alternatively, you can skip the macOS build by adding `--os=lw` below.
```

File: doc/gitian-building.md [hunk 3]
```diff
@@ -124,4 +124,67 @@ Gitian build.
...
+```
+
+Clone your gitian.sigs repo
+```
+git clone https://github.com/$NAME/gitian.sigs
+```
+
+Now, run gitian-build.py script in setup mode. You may get asked to enter your password. 
+```
+./gitian-build.py -V lxc --setup $NAME $VERSION
```
Comment (thephez) [inline doc/gitian-building.md:174]: Why specify `-V lxc`? That's only necessary if someone doesn't have Docker, right?

Comment (PastaPastaPasta) [reply doc/gitian-building.md:174]: It's how I've always done it on ubuntu

**Comment by xdustinface** [comment]
@PastaPastaPasta linter is unhappy

**Comment by knst** [comment]
Closed due to gitian removing: #5756

**Pull request closed without merging**